support/module/packaged/modules.json

```diff
@@ -13,12 +13,12 @@
 			"integrity": "sha512-h1wM7mE/cgpk2n1YOYBNMNBQq5ViLxil6+GRnVqSaFB3LY3ubGSZVkZQ5rzC74anLlNXJVpsH/bP98AQ6zzXbA=="
 		},
 		"ti.map": {
-			"url": "https://github.com/appcelerator-modules/ti.map/releases/download/ios-3.1.0/ti.map-iphone-3.1.0.zip",
-			"integrity": "sha512-CHW1z4RugXToSdBg24UhUTVcg6XL+eY9Q1Org1XeJkuiQsR6DGzywL+MtLZxZw3XflrsfMpyrSNTDKHVmNSQEQ=="
+			"url": "https://github.com/appcelerator-modules/ti.map/releases/download/ios-3.1.2/ti.map-iphone-3.1.2.zip",
+			"integrity": "sha512-t/vHb0hckmfqQmElsRV6mG67m0P+dTe0lXHPYIIX6wAO5thz2kbWjVwgp53hx2IAfJtCsCO9bX15+V5nGZ5DUw=="
 		},
 		"ti.safaridialog": {
-			"url": "https://github.com/appcelerator-modules/ti.safaridialog/releases/download/iOS-1.1.1/ti.safaridialog-iphone-1.1.1.zip",
-			"integrity": "sha512-6g3Qi9RUCl3uf06OnrThm9fpxhcxhtRzyoFRE5PJpenTFN45QYGRZeEQAqgTKcvEB0ardUfGMOyZcA8WoCfWTw=="
+			"url": "https://github.com/appcelerator-modules/ti.safaridialog/releases/download/iOS-2.0.0/ti.safaridialog-iphone-2.0.0.zip",
```

Comment:
Can we include a new major of a module in 8.1.0?

Reply:
Modules have their own release cycle then sdk. So I think it should not be a problem including major of module in 8.1.0. @mukherjee2 @lokeshchdhry  ?

Reply:
@vijaysingh-axway By default we always use the latest version of a module of no explicit version is specified in tiapp.xml. Now, if the ti.safaridialog module contains a breaking API change this will break existing projects.

@ewanharris already opened a ticket for this issue, i just can't find it right now.

Reply:
@ewanharris @janvennemann Restored ti.safaridialog to non-major version.